### DISCUSSION TITLE
Bun v0.5.6

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#2018] [Announcements]
**[Read Bun v0.5.6's release notes on Bun's blog](https://bun.sh/blog/bun-v0.5.6)**

This release is a follow up to [Bun v0.5](https://github.com/oven-sh/bun/releases/tag/bun-v0.5.0)

To install:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade:
```bash
bun upgrade
```

<a href="https://bun.sh/blog/bun-v0.5.6">

<img height="390" src="https://user-images.githubusercontent.com/709451/217699143-6ef0f26a-f8c4-4bad-9aea-d9a8c887b375.png"  />

</a>

## What's Changed
* Fix/Update Typescript types by @gaurishhs in https://github.com/oven-sh/bun/pull/1972
* Implement os.cpus() for Linux by @jwhear in https://github.com/oven-sh/bun/pull/1977
* Bump querystring-es3 by @gaurishhs in https://github.com/oven-sh/bun/pull/1988
* Fix URL of wasi-js by @TerrorJack in https://github.com/oven-sh/bun/pull/1997
* Add types for node:console and node:perf_hooks by @colinhacks in https://github.com/oven-sh/bun/pull/1982
* fix(uWS:capi) uws_req_set_yield typo and uws_res_override_write_offset by @cirospaciari in https://github.com/oven-sh/bun/pull/1996
* Expose Dirent and Stats by @michalwarda in https://github.com/oven-sh/bun/pull/1998
* Fix gaps in types by @colinhacks in https://github.com/oven-sh/bun/pull/2003
* [WIP] fix corner cases with aliases dependencies by @alexlamsl in https://github.com/oven-sh/bun/pull/2000
* [install] assorted fixes & improvements by @alexlamsl in https://github.com/oven-sh/bun/pull/2011
* Fix minor version bug for README.md by @RobViren in https://github.com/oven-sh/bun/pull/2013
* utf16 to utf8 conversion validation by @dylan-conway in https://github.com/oven-sh/bun/pull/2001
* [install] pick `latest` tagged version by default by @alexlamsl in https://github.com/oven-sh/bun/pull/2016

## New Contributors
* @jwhear made their first contribution in https://github.com/oven-sh/bun/pull/1977
* @TerrorJack made their first contribution in https://github.com/oven-sh/bun/pull/1997
* @michalwarda made their first contribution in https://github.com/oven-sh/bun/pull/1998
* @RobViren made their first contribution in https://github.com/oven-sh/bun/pull/2013

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v0.5.5...bun-v0.5.6

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v0.5.6'>Bun v0.5.6</a>.</em>

**Comment by tavaresgerson** [comment]
Ready for production? :heart_eyes:

**Comment by galaxydev55** [comment]
I can do you think
